cli/src/main.rs

```diff
@@ -454,8 +454,17 @@ fn main() -> Result<()> {
 
 /// Adds the CLI runtime to the context with default options.
 fn add_runtime(context: &mut Context) {
-    boa_runtime::register(context, boa_runtime::RegisterOptions::new())
-        .expect("should not fail while registering the runtime");
+    boa_runtime::register(
+        (
+            boa_runtime::extensions::ConsoleExtension::default(),
+            boa_runtime::extensions::FetchExtension(
+                boa_runtime::fetch::BlockingReqwestFetcher::default(),
+            ),
```

Comment:
Important: The CLI uses `BlockingReqwestFetcher` and fails if the default feature are disabled (see build CI), this should be featured flagged that enables the `boa_runtime/fetch` and `boa_runtime/reqwest`
```suggestion
            boa_runtime::extensions::FetchExtension(
                boa_runtime::fetch::BlockingReqwestFetcher::default(),
            ),
```

Reply:
Yeah added a `fetch` feature to cli that is by default but removing the default features `fetch` won't be defined.